### PR TITLE
build(deps): Pin dependency versions

### PR DESCRIPTION
**Pull request by mojito317**
## PR summary

In preparation for removing the "pre-release" flag, we should stop using version ranges so that installed versions use a specific set of dependencies.

The versions I pinned are from the current `package.lock.json` file: https://github.com/IBM/cloudant-node-sdk/blob/3396b2482b8aa5309e754cba48b282ae5d8fc8d6/package-lock.json

## PR Checklist

Please make sure that your PR fulfills the following requirements:

- [x] The commit message follows the
[Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [x] Build/CI related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

## What is the current behavior?
We are using version ranges.

## What is the new behavior?
We are using pinned versions.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,37 +34,37 @@
...
+    "eslint-plugin-header": "3.1.1",
+    "eslint-plugin-import": "2.24.2",
+    "eslint-plugin-node": "11.1.0",
+    "eslint-plugin-prettier": "4.0.0",
+    "jest": "26.6.3",
+    "jest-junit": "12.2.0",
+    "prettier": "2.3.2",
+    "semantic-release": "17.4.7",
+    "sinon": "11.1.2",
+    "tough-cookie": "4.0.0",
```
Comment (ricellis) [inline package.json:63]: Given our requirement for this in the session authenticator I think we should move this into `dependencies`

**Review by ricellis — APPROVED** [review]
I'm +1, but I think we should move `tough-cookie`

File: package.json (hunk 1 above)
Comment (mojito317) [reply package.json:63]: Oh wow, I thought I did this, maybe the squash was not worked the way I expected. Thanks!

**Review by vmatyusGitHub — APPROVED** [review]

**Comment by mojito317** [comment]
Thanks for the reviews @ricellis and @vmatyusGitHub! I'm merging this!

**Pull request merged**